experimental/packages/opentelemetry-exporter-metrics-otlp-http/src/transformMetrics.ts

```diff
@@ -15,7 +15,7 @@
  */
 
 import { SpanAttributes, HrTime } from '@opentelemetry/api';
-import { Labels, ValueType } from '@opentelemetry/api-metrics';
+import { Attributes as Labels, ValueType } from '@opentelemetry/api-metrics';
```

Comment:
Newbie question. On line 30-39 it would not change?

Reply:
I left it as `Labels` in this case, since its using the types defined for the 0.6.0 proto (until https://github.com/open-telemetry/opentelemetry-js/issues/2586 is done, probably). The field is called `labels` in the TS representation of the protobuf, so I kept it that way.